domain/domain.template.yml

```diff
@@ -123,6 +123,12 @@ Resources:
           Type: CNAME
           TTL: 172800
 
+        - Name: !Ref Domain
+          Type: TXT
+          ResourceRecords:
+          - "\"google-site-verification=29R-qq6QJ-1kNjZl5HDOxwbQkPihE3Xs7nbRwa9FdGY\"" # Added for Huw's Google Search Console setup.
```

Comment:
I do think this format is a bit strange, not sure why I couldn't just have:
```
- "google-site-verification=29R-qq6QJ-1kNjZl5HDOxwbQkPihE3Xs7nbRwa9FdGY" # Added for Huw's Google Search Console setup.
```
But it follows what @paroxp [did here](https://github.com/govuk-one-login/domains/pull/84/files#diff-002bffc600096340e660c75c74ca4524767f7f9d69ed4a95f4bf2e1e65e625acR599), and I suspect there's a really good reason for it.

Reply:
Yeah, the CloudFormation didn't lint without it..